src/utils/colonyMotions.ts

```diff
@@ -30,47 +30,47 @@ export enum MotionVote {
 
 const MSG = defineMessage({
   stakedTag: {
-    id: 'dashboard.ActionsPage.stakedTag',
+    id: 'common.ColonyHome.ActionsPage.stakedTag',
```

Comment:
I proposed the following change again in simple decisions, since these messages aren't being defined inside a component (and at least one of which is used in DraftDecisionItem and not Action Page), they could use the other style of message ids: https://github.com/JoinColony/colonyCDapp/pull/201/files#r1085082413

Reply:
I'll revert this change then, and let the changes you did on Simple Decision take over eventually